parcel.go

```diff
@@ -13,48 +14,79 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	parcelAdd, err := s.db.Exec(
+		"insert into parcel (client, status, address, created_at) values (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, fmt.Errorf("произошла ошибка добавления новой записи в базу данных, %w\n", err)
```

Comment:
Молодец, что оборачиваешь ошибки, только не нужно делать перенос, это уже при выводе можно добавить форматирование
```suggestion
		return 0, fmt.Errorf("произошла ошибка добавления новой записи в базу данных, %w", err)
```

Reply:
Там на самом деле это можно оборачивать до бесконечности и оно по сути просто склеивает в одну строку, так что лучше без переноса